### ISSUE TITLE
Buildings appear at a higher a zoom level

### ISSUE THREAD

**Issue by kepta** [openstreetmap/iD#4269]
Don't know if this is a regression, I see buildings appear and then disappear. 
Here a recording of this behaviour.

![out](https://user-images.githubusercontent.com/6966254/29702172-9e5c6586-898d-11e7-809f-1c5459f0360d.gif)

**Comment by bhousel** [comment]
yes this is by design - when the buildings or points exceed a certain threshold, they are hidden and iD shows this little blue message down in the corner:

<img width="416" alt="screenshot 2017-08-25 09 28 22" src="https://user-images.githubusercontent.com/38784/29716042-d49fe0a2-8977-11e7-9484-5ec810dcd7bb.png">

**Comment by pnorman** [comment]
Do we still need to hide for performance reasons? A lot has changed since that was introduced, and as we can here, it can be confusing to have features disappear and appear when panning without zooming.

**Comment by kepta** [comment]
I was thinking of ways we can avoid rendering it in the first place, for that we might need the feature count before hand. I fear this is dependant on network and hence not in our hands...

**Comment by giblet22** [comment]
We see some issues around this with new mappers. If they don't see the notification, I've seen some buildings mapped 4 times in more dense areas (where it's still clear to trace on the imagery). If I'm leading an event, it's certainly on me to make sure they're aware of the notice, but could an additional solution be making it more prominent? Or maybe that it blinks to draw some attention to it?
